### DISCUSSION TITLE
Tap on a clip to paste it in the text box.

### DISCUSSION THREAD

**Discussion by Albuflasa107** [community/community#37418] [Other Feature Feedback, Questions, & Ideas]
Jaber_s33d@icould.com

**Comment by gyami5385** [comment]
gyami5385@gmail.com

**Comment by ebndev** [comment]
Hi @Albuflasa107, thanks for being a part of the GitHub Community!

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" as GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. Any repeat and future violations may result in a temporary or indefinite block from the Community. 

We appreciate your understanding.
